src/Diagnostics.DataProviders/KustoClient.cs

```diff
@@ -79,7 +79,14 @@ public KustoClient(KustoDataProviderConfiguration configuration)
 
             DataTableResponseObjectCollection dataSet = JsonConvert.DeserializeObject<DataTableResponseObjectCollection>(content);
 
-            return dataSet.Tables.FirstOrDefault().ToDataTable();
+            if (dataSet.Tables == null)
```

Comment:
nitpick : maybe just check for dataSet null and also dataSet.Tables.Any() to make it more solid and less error prone.

Reply:
Doing `dataSet.Tables.Any()` wont work in this case because it will throw NullReferenceException.